### ISSUE TITLE
Better error when wrong file name

### ISSUE THREAD

**Issue by jimaek** [jsdelivr/bot#41]
A user added info.txt instead of info.ini and he got this error
```
Might be nothing but I have some concerns about the files in your submission...

    string indices must be integers, not str
```

We need to add a better error explaining better what is wrong.

**Comment by megawac** [comment]
Can you link me the relevant PR this didn't work for

**Comment by jimaek** [comment]
https://github.com/jsdelivr/jsdelivr/pull/4523

**Comment by megawac** [comment]
Fixed in c0e28b527, it was a simple bug
